### ISSUE TITLE
Smooth and Sharp edge tiles switch with camera rotation

### ISSUE THREAD

**Issue by ShenKitchen** [OpenRCT2/OpenRCT2#12058]
**OS:** [ Windows 10]
**Version:** [v0.2.6-994-g681b8d793]
**Commit/Build:** [681b8d7]

**Screenshots / Video:**

![image](https://user-images.githubusercontent.com/67436450/85777801-cdc94e00-b6d6-11ea-8a50-93148b9b44be.png)

**Save game:**

**Comment by James103** [comment]
Are you leaving the relevant setting ("Landscape Smoothing", see also "Gridlines on landscape") at the same value in your tests (so leaving it on while you switch camera rotation)? For example, turn landscape smoothing on, observe the landscape detail, rotate the camera, observe landscape detail again without changing the setting.

Also, what is the exact OpenRCT2 version you are using, and can you reproduce this in RCT1/2 vanilla?

During my tests (6c3c857), regardless of the values of the two landscape settings and the camera rotation, I could not find any such inconsistencies.

**Comment by Gymnasiast** [comment]
@ShenKitchen Can you fill in the actuall version and build hash? You left the placeholder data in there.

**Comment by e-foley** [comment]
Possibly related to #2333?

**Comment by Gymnasiast** [comment]
@e-foley No, that's another issue. #2333 is only about smoothing within one tile, while this is about smoothing between tiles.

**Comment by duncanspumpkin** [comment]
Can you check again on latest develop if this is still happening

**Comment by Gymnasiast** [comment]
Assuming that this is indeed a duplicate of #2333 after all.
